src/gpu/intel/jit/ir/problem.hpp

```diff
@@ -124,6 +124,11 @@ class pvar_map_t {
             operator[](k) = value;
     }
 
+    pvar_map_t(const std::initializer_list<std::pair<pvar_t, ValueT>> &keys) {
```

Comment:
Feel free to drop this patch. It seems irrelevant to the DSL and more closely tied to the IR GEMM implementation.